src/components/slider/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+
+import cn from 'lib/classnames';
+
+import * as SliderPrimitive from '@radix-ui/react-slider';
+
+const Slider = ({ defaultValue, max = 1, step = 0.1, onValueChange, className }) => {
```

Comment:
Not sure about this.

const Slider = forwardRef<
  ElementRef<typeof SliderPrimitive.Root>,
  ComponentPropsWithoutRef<typeof SliderPrimitive.Root>
>(({ className, ...props }, ref) => (
  <SliderPrimitive.Root
    ref={ref}
    className={cn('relative flex h-4 w-full touch-none select-none items-center': true,
          [className]: !!className)}
    {...props}
  >

Reply:
Think this is inside radix ui component

Reply:
use shadcn https://ui.shadcn.com/docs/components/slider